onboard/main.cpp

```diff
@@ -32,8 +34,11 @@ enum STATES{
   WAIT
 };
 int state = STATES::WAIT;
-int currLocalJoint = 0;
+int currJoint = 0;
 volatile unsigned long int timer = 0;
+volatile boolean PIDflag = true; //flag based on timer for PID control
+unsigned long int watchdog_timer; //for live control
```

Comment:
Watchdog should be based on a heartbeat unrelated to live control. It should always be listening/responding to heartbeat so both sides know if something fails.